friends/src/components/Form.js

```diff
@@ -0,0 +1,52 @@
+import React from "react";
```

Comment:
Love this form component. Refactored it to immediately move user's cursor to the name Input for easy editing. Also enabled the user to submit the form by pressing ENTER key anywhere in the form component and the ability to exit the form component when USER presses ESCAPE key. Awesome job. Very impressive work. 🔥 
![image](https://user-images.githubusercontent.com/26180499/47562209-a6ebc080-d8d2-11e8-9920-9f077ec868b8.png)